packages/components/select/src/select-icon.tsx

```diff
@@ -50,7 +50,7 @@ export const SelectIcon = forwardRef<SelectIconProps, "div">(
         __css={css}
         {...rest}
       >
-        {isValidElement(children) ? cloneChildren : <ChevronIcon />}
+        {isValidElement(children) ? cloneChildren : <ChevronDownIcon />}
```

Comment:
```suggestion
        {isValidElement(children) ? cloneChildren : <ChevronDownIcon fontSize={'md'} />}
```

Reply:
@hirotomoyamada 
Thank you for your comment. I understand that you are saying the icon I replaced is bigger than the original one. If so, I am planning to make the changes as shown above. Does that work for you?